addons/pos_restaurant/static/src/app/floor_screen/table.js

```diff
@@ -70,7 +77,10 @@ export class Table extends Component {
     }
     get orderCount() {
         const table = this.props.table;
-        return table.order_count !== undefined
+        const orderOnTable = this.env.pos
```

Comment:
rename `orderOnTable` to `numOrdersOnTable`

Moreover,
- you extracted this computation but you didn't replace the same call below. (Check the return value.)
- and, `this.pos.env` doesn't work anymore. Use `this.pos.globalState`.